virttest/qemu_vm.py

```diff
@@ -1879,6 +1883,8 @@ def sort_key(dev):
                                                      backend="backend")
                 pvpanic_dev.set_param("id", utils_misc.generate_random_id(),
                                       dynamic=True)
+                if pvpanic == 'pvpanic-pci':
```

Comment:
Since `pvpanic-pci` is a PCI device, so I think you should use `qdevices.QDevice` to add this device for aarch64, it will add `bus` and `addr` properties based on `parent_bus`.

Reply:
thanks a lot, update later.